cristin-import/src/test/java/no/unit/nva/cristin/lambda/CristinEntryEventConsumerTest.java

```diff
@@ -134,221 +136,226 @@ void shouldReturnAnNvaPublicationEntryWhenInputIsEventWithCristinResult() throws
         handler.handleRequest(eventReference, outputStream, CONTEXT);
         var json = outputStream.toString();
         var actualPublication = eventHandlerObjectMapper.readValue(json, Publication.class);
-        
+
         var expectedPublication = cristinObject.toPublication();
         injectValuesThatAreCreatedWhenSavingInDynamo(actualPublication, expectedPublication);
-        
+
         assertThat(actualPublication, is(equalTo(expectedPublication)));
         assertThat(actualPublication.getStatus(), is(equalTo(PublicationStatus.PUBLISHED)));
     }
-    
+
+    @Test
+    void shouldStoreNvaPuclicationIdAndCristinIdInS3() throws IOException {
```

Comment:
Publication

Reply:
I guess it is "shouldPersistCristinIdInFileNamedWithPublicationIdentifier"

Reply:
Fikset